tests/yaml/capsnocont.yaml

```diff
@@ -14,26 +14,3 @@ table: |
 tests:
   - [UNDER et, ⠠⠠⠥⠝⠙⠑⠗ ⠬]
   - ["ABC Under et", "⠠⠠⠁⠃⠉ ⠠⠟ ⠬"]
-
-# test capsnocont when no begcapsword indicator is defined
-# every cap will be marked with capsletter
-table: |
-  include tables/spaces.uti
-  include tables/latinLetterDef6Dots.uti
-  punctuation , 6
-  capsletter 6
-  # No capsword rules
-  # begcapsword 6-6
-  # endcapsword 6
-  capsnocont
-  word foo 124
-  word bar 12
-  partword foo 1346
-  partword bar 13456
-
-tests:
-  - ["foo bar foobar", "f b xy"]
-  - ["Foo Bar Foobar", ",f ,b ,xy"]
-  - ["FOO BAR FOOBAR", ",f,o,o ,b,a,r ,f,o,o,b,a,r"]
-  - ["TXT", ",t,x,t"]
-  - ["ABC Foobar", ",a,b,c ,xy"]
```

Comment:
Where di these tests go? I couldn't find them in the new `emphasis.yaml`.

Reply:
This was replaced with https://github.com/liblouis/liblouis/blob/ff9bf4cfd61a3c24c7d72789b55441f7eb290a3b/tests/yaml/emphasis.yaml#L108-L122. Since `capsnocont` has nothing to do with it anymore, this exact test didn't make sense anymore.

Reply:
OK